docs/DeveloperGuide.md

```diff
@@ -309,18 +338,19 @@ The following activity diagram summarizes what happens when a user enters an `ed
 
 ### Marking all Rooms as Unclean
 
-* In GuestBook, you can mark all the rooms as unclean. This feature was added to make it easier for the hotel to transit to a new working day, as they would usually have to clean all the rooms when a new day starts.
-#### Implementation
+In GuestBook, you can mark all the rooms as unclean. This feature was added to make it easier for the hotel to transit to a new working day, as they would usually have to clean all the rooms when a new day starts.
```

Comment:
Should we standardize either the user or you?

Reply:
Yes, fixed.